utils.go

```diff
@@ -14,14 +33,150 @@ type Storage struct {
 
 // String implements Storager.String
 func (s *Storage) String() string {
-	panic("implement me")
+	return fmt.Sprintf("Storager ftp {Name: %s, User: %s, WorkDir: %s}", s.name, s.user, s.workDir)
 }
 
 // NewStorager will create Storager only.
 func NewStorager(pairs ...types.Pair) (types.Storager, error) {
-	panic("implement me")
+	return newStoragerWithFTPClient(pairs...)
+}
+
+func newStoragerWithFTPClient(pairs ...types.Pair) (store *Storage, err error) {
```

Comment:
```suggestion
func newStorager(pairs ...types.Pair) (store *Storage, err error) {
```

There's no other `newStorager` here, maybe we can simplify the function name?